### PR TITLE
Desktop view of my portfolio website

### PR DESCRIPTION
**Pull request by citec-47**
**##DESKTOP VERSION OF PORTFOLIO WEBSITE WEEK TWO✔✔✔**
#Below are the general requirement used for developing this project✔
##General requirements🧾🧾
-Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).✔
-Make sure that you used correct ([Github flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git--github/articles/github_flow.md).✔
✔Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum- 
  transversal---##skills/blob/main/documentation/articles/professional_repo_rules.md).
 #HTML/CSS requirements
-Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-- 
 -css/blob/main/articles/html_css_best_practices.md).
##Project requirements✔✔
-You need to make sure that your desktop version looks like the one in the Figma design template that you chose. We include a screenshot of Template 1 below for reference, but you should use the template that you chose.

#Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px).

#In order to lay out the elements on the page you should use Grid and Flexbox.✔

### CODE REVIEW

File: portfolio-Destktop-versionportfolio/index.html [hunk 1]
```diff
@@ -64,98 +70,83 @@ <h1 class="headerText">Multi-Post Stories</h1>
...
-  personalized reads; no accounts 
-  or sign-ups required. has been the
-   industry's standard</p>
-   <ul class="button">
-    <li>html</li>
-    <li>bootstrap</li>
-    <li>Ruby</li>
-  </ul>
-  <button type="button" class="see">see Project</button>
+<div class="professional">
```
Comment (topeogunleye) [inline portfolio-Destktop-versionportfolio/index.html:81]: - [ ] I kindly suggest that on when the project is hovered on with the mouse the description and dark opacity disappear and a See Project button appears to the bottom of the image and takes the whole image width 👍🏿
 
 - In the projects section the design with a cursor pointer is representing the state of the individual project on hover. We are supposed to make it such that the dark opacity disappears when hovering over it and the image with the `See Project` button appears. Please update them accordingly.
 
 
| Normal State  | Hover State |
| ------------- | ------------- |
| 	  |   |

File: portfolio-Destktop-versionportfolio/index.html [hunk 2]
```diff
@@ -64,98 +70,83 @@ <h1 class="headerText">Multi-Post Stories</h1>
...
+<button type="button" class="lord"> <a class="work-button" href="https://github.com/citec-47/portfolio/pull/5">See Project</a>
+</div>
```
Comment (topeogunleye) [inline portfolio-Destktop-versionportfolio/index.html:83]: - [ ] I kindly suggest that the description of each of these projects should be aligned to the right and bottom of the image.


|  |  |
| ------------- | ------------- |
| Your Project | image  |
| Figma Design |  image|

File: portfolio-Destktop-versionportfolio/index.html [hunk 3]
```diff
@@ -164,11 +155,10 @@ <h1 class="headerText">Multi-Post Stories</h1>
...
-    <p>Hello I’m a software developer! I can help you build a product , 
-      feature or website Look through some of my work and experience! If you
-       like what you see and have a project you need coded, don’t hestiate to contact me.
-      </p>
-    <button type="button">Get my Resume</button>
+    <h5>Hello I’m a software developer! I can help you build a product , feature or 
+      website Look through some of my work and experience! If you like what you see and have a project you need coded,
+       don’t hestiate to contact me.</h5>
+       <button type="button" class="get">Get my Resume</button>
    </div>
```
Comment (topeogunleye) [inline portfolio-Destktop-versionportfolio/index.html:162]: - [ ] I kindly suggest that your Language, Frameworks, and skills should be aligned properly.

image

File: portfolio-Destktop-versionportfolio/index.html [hunk 4]
```diff
@@ -198,39 +188,35 @@ <h1 class="headerText">Multi-Post Stories</h1>
...
     <li><a href="#"><img alt="linkedin-icon" src="./image/Disabled.png"></a></li>
     <li><a href="#"><img alt="media-icon" src="./image/Vector1.png"></a></li>
     <li><a href="#"><img alt="media-icon" src="./image/Vector.png.png"></a></li>
     <li><a href="#"><img alt="meduim-icon" src="./image/Vector2.png"></a></li>
   </ul>
- </footer>
+</footer>
 
   </body>
 </html>
```
Comment (topeogunleye) [inline portfolio-Destktop-versionportfolio/index.html:222]: - [ ] I kindly suggest that you remove the horizontal scroll bar as the bottom that is a result of your webpage not being responsive enough.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @citec-47, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have running linters. Please ensure that your linters are running before you submit for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by citec-47** [comment]
Good morning Sir, please i really do need your guidance on how to fixed the
hover image issue.This is my second week in week and I don't want to repeat
again. Just a guide on how to do it will really be of a great help to me

**Comment by citec-47** [comment]
**#AJUSTMENT OF HOVER EFFECTS✔✔✔**
#Changes Required🌳
-Hover Effects✔
-Alignment of items on About page to take a straight patten✔
##Solution🌳
-Hover Effect Adjusted✔
-About page items adjusted✔
-Hover button effect has been adjusted✔

🙏🙏🙏Thank you so much for reviewing my codes and please sir you can help me with links that can help me learn more.

**Review by levy002 — COMMENTED** [review]
## Hello @citec-47 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't run linters Kindly fix linter errors and submit again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by citec-47** [comment]
#Arrangement Of My Design To Match The One In Figma✔✔

🙏🙏Thanks For Reviewing my codes i have learn a lot and i really do appreciate

**Comment by citec-47** [comment]
#DESKTOP AND MORBILE AJUSTMENT✔✔
#Desktop View  ✔


##MOBILE VIEW✔✔

File: style.css [hunk 5]
```diff
@@ -471,24 +649,440 @@ form textarea {
...
-  display: flex;
-  width: 10rem;
-  justify-content: space-between;
-  align-items: center;
-  list-style: none;
-  gap: 1rem;
-  margin-left: 4rem;
-}
+@media only screen and (min-width: 768px) {
+  .john {
```
Comment (joskalenda) [inline style.css:668]: - [ ] Seeing that you have Updated the linter checks, now let's dive into the design checking, I will drop the record below to show you how broken the app is, you have settled your app media query screen from 768 px but the website is not responsive at all.



- [ ] The second thing is that the project button z-index is high and it stands on the header, make sure you fix it.

File: index.html [hunk 6]
```diff
@@ -64,111 +70,150 @@ <h1 class="headerText">Multi-Post Stories</h1>
...
+    </h1>
+    <h2>A daily selection of privately personalized reads;
+       no accounts or sign-ups required.
+       has been the industry's standard</h2>
+       <ul class="buttons">
+        <li>html</li>
+        <li>bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+    </div>
```
Comment (joskalenda) [inline index.html:93]: - [ ] The requirement says to keep your mobile version updated and functional regarding the Figma design, by that, allow me to point out this issue; your card projects are not well centered, if would please make some changes to make the spaces on the right and left be equal.

File: index.html [hunk 7]
```diff
@@ -198,39 +243,35 @@ <h1 class="headerText">Multi-Post Stories</h1>
...
+
+ <form action="https://formspree.io/f/meqwogjz" method="post">
+  <h2 class="About-header">
     I'm always interested in hearing about new projects,
      so if you'd like to chat please get in touch.
-   </h1>
- <form action="https://formspree.io/f/meqwogjz" method="post">
-  <ul>
+  </h2>
+  <ul class="form">
```
Comment (joskalenda) [inline index.html:252]: - [ ] Despite the broken style, please make sure you respect the original design `padding, margin, colors, font-weight, and font-size`, currently you are missing some properties, like this on the image bellow, if I believe my eyes, the font-eight used in your text is not the same as the one in the correct design, please fix that.

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @citec-47 , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]... ✅ Green linters
- [ ].. ✅ Great readme file

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 7 above)
Comment (citec-47) [reply index.html:252]: @joskalenda BRO HAVE BEEN ON THIS SINCE LAST WEEK AND TO TELL YOU THE TRUTH THIS IS MY LAST REVIEW FOR TODAY

**Comment by citec-47** [comment]
#ARRANGEMENT OF BROKEN BACKGROUND IMAGES

**Comment by citec-47** [comment]
##THIS IS MY LAST REVIEW FOR THE WEEK PLEASE SIR TAKE YOUR TIME AM BEGGING BECAUSE IT'S VERY DISCOURAGING TO SPEND SLEEPLESS NIGTH AND STILL NOT GET APPROVE FOR TWO WEEKS😢

File: style.css [hunk 8]
```diff
@@ -206,62 +281,163 @@ body {
...
+  background-image: url("./image/ccc.png.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: auto;
+  padding-top: 90px;
+  margin-left: 0;
+}
+
+.professional4 {
+  display: flex;
```
Comment (Owusu-Desmond) [inline style.css:340]: - [ ] While making changes requested by the previous reviewers, Please kindly make sure your project is responsive on desktop and mobile screens. But your project has a horizontal scroll that shows your project is not responsive so please make sure you adjust the style of your application to remove the horizontal scroll. I have recorded a video highlighting the change you have to work on as you can below. :pray:

https://user-images.githubusercontent.com/92303891/215294542-9ca5ddc4-374c-4607-844b-7eeb79a4212b.mp4

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @citec-47 , :wave: 
 
You've done well implementing some of the requested changes, but there are still some changes you still need to work on before you move on to the next project but you are almost there.



 ## Highlights :+1: 
:heavy_check_mark: Good commit history.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Your project is well documented


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 9]
```diff
@@ -628,6 +628,10 @@ form input {
   padding: 14px 14px 14px 14px;
 }
 
+body {
+  overflow: hidden;
+}
+
```
Comment (GedeonTS) [inline style.css]: - [ ]  As suggested by the previous reviewers kindly fix the responsiveness of your website, here I see you have set overflow to be hidden, this introduces a bug in your website as it is preventing the user from scrolling from top to bottom. :recycle:

kindly remove this property and add responsivess as requested by the previous reviewer [here](https://github.com/citec-47/PORTFOILIO-WEEK-TWO/pull/2#discussion_r1089827310)

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :
Hi @citec-47 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights :tophat: :+1: 
- No linter errors :+1: 
- Good use of GitHub flow :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions :ringed_planet: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sinansevgi — COMMENTED** [review]
Hello @citec-47 👋,

_This is an additional review as a result of the second opinion request._

It is imperative to consider the user experience while designing a website. The overflow property is set to hidden, which prevents the users from scrolling from top to bottom, creating a bug in the website.

To resolve this issue, it is recommended to remove the "overflow: hidden" property and replace it with a more suitable solution that allows for smooth scrolling. This will provide a better user experience, as users will be able to navigate the website without any hindrances.

Additionally, implementing responsiveness is a crucial requirement of the project. By using CSS media queries, the website can adapt to different screen sizes, providing a tailored experience for both desktop and mobile users. This will ensure that the website meets the project requirements and provides a seamless experience for the users across all devices.

In conclusion, taking these steps will greatly improve the overall user experience, meet project requirements, and make the website more accessible and user-friendly. Please implement these changes requested by the @GedeonTS and request another review.

Keep rocking 🚀!
